helpers.js

```diff
@@ -255,7 +255,7 @@ if (local) {
   before(function () {
     // Note: you need to run from the root of the project
     // TODO: path.resolve
-    server = new SeleniumServer('./vendor/selenium-server-standalone-2.41.0.jar', {
```

Comment:
@hmgibson23 I think this is the magic that was causing your `Server not started` issue.